src/xunit.netcore.extensions/TargetFrameworkMoniker.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit.Sdk;
+
+namespace Xunit
+{
+    [Flags]
+    public enum TargetFrameworkMoniker
```

Comment:
`PlatformID` really should be plural as well, to follow the design guidelines. Then it wouldn't conflict with `System.PlatformID`.